src/ontology/uberon-edit.obo

```diff
@@ -224055,6 +224055,14 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
 relationship: part_of UBERON:0006767 ! head of femur
 property_value: dcterms-date "2023-05-30T06:34:18Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480063
+name: dorsal iris
+def: "The dorsal part of the iris. This region is involved in lens regeneration in urodeles (newts and salamanders) in a process named Wolffian lens regeneration. In these species, lens regeneration proceeds through two steps. The first step involves the entire circumference of the iris pigmented epithelium. The second step is confined to the pigmented epithelial cells located along the dorsal rim of the iris and involves dorsal-specific activation of the Wnt signaling pathway." [PMID:17030116, PMID:29579253]
```

Comment:
Consider keeping the first two sentences in the text definition and relegating the rest to a comment.

Reply:
Also consider if a new GO term 'Wolffian lens regeneration' is warranted.

Reply:
Indeed this term does not exist in GO. Should I make a NTR to GO?